socialNetwork/src/UserTimelineService/UserTimelineService.cpp

```diff
@@ -80,6 +81,11 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+  if (redis_replica_config_flag && (redis_cluster_config_flag || redis_cluster_flag)) {
+      LOG(error) << "Cannot start with enabled redis cluster and redis replica flags both";
```

Comment:
fix message - `Can't start service when Redis Cluster and Redis Replica are enabled at the same time`